scAR/integration_tests/scar_integration_test.py

```diff
@@ -0,0 +1,26 @@
+from scAR import model
+import pandas as pd
+import sys
+import os
+
+raw_count = pd.read_pickle('../test/raw_counts.pickle')
+empty_profile = pd.read_pickle('../test/ambient_profile.pickle')
+expected_output = pd.read_pickle('../test/output_assignment.pickle')
+
+scarObj = model(raw_count=raw_count.values,
+                empty_profile=empty_profile,
+                scRNAseq_tech='CROPseq')
+
+scarObj.train(epochs=40,
+              batch_size=64,)
+
+scarObj.inference()
+
+status = scarObj.feature_assignment.equals(expected_output)
+
+if status:
+    sys.stdout.write(f"Successful integration test.{os.linesep}")
+    sys.exit(0)
```

Comment:
Could we raise exceptions instead?

Reply:
I made some adjustments @mr-nvs 
Is that better?

Reply:
Yep, I think so!